### ISSUE TITLE
Considerations around Steering and potential Conflict of Interest 

### ISSUE THREAD

**Issue by mrbobbytables** [kubernetes/steering#283]
The individuals of the Steering Committee are elected with the knowledge and trust that they will **represent the project** over their employer, personal interests, etc. We have [maximal representation](https://github.com/kubernetes/steering/blob/main/elections.md#maximal-representation) as a safeguard for employer influence, and a note regarding [Abstention](https://github.com/kubernetes/steering/blob/main/charter.md#abstention) in our voting guidelines that acts as a general catch-catch all.

This covers the vast majority of situations as it is expected that steering members with a CoI would abstain from a vote. However there are situations where a more direct acknowledgement of the conflict would be beneficial to avoid any potential biases. e.g. there is a direct issue with an employer or the steering member is a SIG lead for a SIG that has had an issued raised about its leadership.

Should a CoI statement be added to the charter? If so, would it change how we vote? or would it be considered an abstention?

IMO - even if it is determined that no changes are needed, I'd lean towards moving the section regarding [maximal representation](https://github.com/kubernetes/steering/blob/main/elections.md#maximal-representation) from the [election doc](https://github.com/kubernetes/steering/blob/main/elections.md) to the charter. That might be a separate issue to discuss, but I think adding a note about it in the general steering composition section would be good move.

/comittee steering

**Comment by mrbobbytables** [comment]
To be explicit about a current issue; as of last week, I am now an employee of the CNCF. 
We DO have [policies in place around maximal representation](https://github.com/kubernetes/steering/blob/main/elections.md#maximal-representation), and my employer does not technically run afoul of that. However, there is potentially a greater level of CoI as there is a direct project-level relationship with the CNCF.

In addition to maximal representation policy, we do have some guidance for [voting abstention](https://github.com/kubernetes/steering/blob/main/charter.md#abstention) or as a worst case scenario a [no confidence vote](https://github.com/kubernetes/steering/blob/main/charter.md#no-confidence).

Is this enough? should we make a CoI policy more explicit? or some policy around CNCF employment? I am also happy to bow out of the discussion until there is some sort of consensus.

**Comment by cblecker** [comment]
_Disclaimer: While I do sit on the CNCF Governing Board as a representative for the Kubernetes project, the following represents my thoughts as a community member and not that of the foundation or it's board._

Individuals elected to project leadership roles such as the Steering Committee are just that -- individuals. We don't represent our employers. This value is so important to us, we enshrine it in our [values document](https://github.com/kubernetes/community/blob/master/values.md#community-over-product-or-company).

It's not that our employers don't have a vested interest in the project; they do and should. But we as individuals agree to set those interests aside when making leadership decisions for the project to ensure that we are making decisions for the good of the community and project as a whole, and not to advance the interests of any specific employer.

To the specifics at hand here:
- In accordance with our values, I trust the members of the Steering Committee to be able to set the interests of their employer aside and represent the community.
- In the case that an issue comes up related to their employer where they are unable to avoid a direct or perceived conflict of interest, I trust that the members of the Steering Committee would abstain on that issue no matter who the employer is.
- In the case that the other the members of the Steering Committee feel differently on an issue than a specific member does, I trust that each member will vote their conscious. We have an explicit maximal representation policy to ensure that no employer (the CNCF included) can force a vote to go a certain way.
- Lastly, in either the course of working at a company, or leading an open source project, there may be times where someone has knowledge of non-public information that shouldn't be shared outside of the context in which it was learned. For example, internal company information around future plans, or project information such as private Steering Committee discussions. In the case employer information is improperly shared with the project, I expect this would be investigated by said employer and the person involved would face disciplinary consequences. And in the case that private project information is improperly shared with folks not on the Steering Committee, I would personally view that breach of trust very seriously and the Steering Committee has mechanisms (up to and including a vote of no confidence) in order to repair such a breach.

Basically, the conclusion I personally arrived to in my head is that the mechanisms we have seem to cover all the bases here around potential conflicts. While the relationship between the project and the CNCF may feel unique in comparison to other employers, the test of a good/just/fair policy is that even when applied to novel situations, the policy holds up to scrutiny.

**Comment by BenTheElder** [comment]
> Individuals elected to project leadership roles such as the Steering Committee are just that -- individuals. We don't represent our employers. This value is so important to us, we enshrine it in our [values document](https://github.com/kubernetes/community/blob/master/values.md#community-over-product-or-company).

This. And as mentioned repeatedly above, aside from abstention in any somehow dubious topic, in the case that we somehow fail to uphold this, we structurally have maximal employer representation and no confidence mechanisms to make it impossible for any one employer to throw a steering vote anyhow.

Working for the CNCF is a new one for sure, but we already set out to prevent employers from forcing votes and we already expect that our members are democratically elected by the community to uphold our values, "Community over product or company" is very core to the project to me and I think to most, if not all, of the voters.

If there's any gaps in the governance we should address them, but I don't think there are any in this particular case.
I think we've already been quite proactive and forward thinking on this subject, the foundation and steering exist in no small part to ensure that ~neutral space.

I personally don't even see employers forcing steering votes as particularly likely, but just in case and to ensure if nothing else that there's no room to even suspect that such a subversion happened, we have the maximal representation (and a sufficiently large body) and no confidence governance rules in place already.

> IMO - even if it is determined that no changes are needed, I'd lean towards moving the section regarding [maximal representation](https://github.com/kubernetes/steering/blob/main/elections.md#maximal-representation) from the [election doc](https://github.com/kubernetes/steering/blob/main/elections.md) to the charter. That might be a separate issue to discuss, but I think adding a note about it in the general steering composition section would be good move.

I would compromise: I think it's important that the specific handling of this is maintained in the election doc, but we should mention it in the general charter with a link to the elections subsection for the exacting details because it's also important that we highlight this This whole ecosystem thrives off of working together across organizations and I think it's vital that we make this visible.

**Comment by BenTheElder** [comment]
> We could add a statement to our code of conduct that we expect people to recuse / abstain when there is a conflict of interest, and that not doing so isn't acceptable.

We can suggest it, though we don't own this document. Currently that document discuss conflict of interest *as it applies to enforcing the code of conduct*

Quoting TLDR suggested follow-up from myself: https://github.com/kubernetes/steering/issues/283#issuecomment-2161848654
> I would compromise: I think it's important that the specific handling of this is maintained in the election doc, but we should mention it in the general charter with a link to the elections subsection for the exacting details because it's also important that we highlight this This whole ecosystem thrives off of working together across organizations and I think it's vital that we make this visible.

Though again, I generally concur with https://github.com/kubernetes/steering/issues/283#issuecomment-2161804183
I think we have pretty robust mechanisms in place to defend from this as-is. The 7 person committee with majority voting rules with at most 2 from any organization means that practically an organization could not hijack steering behavior even if they are able to "strong arm" members or members actively maliciously behaved. This is in place despite generally assuming and expecting the best from our community at large and especially elected committee members.

@kubernetes/steering-committee, do the rest of you have any objections or alternate thoughts?

**Comment by soltysh** [comment]
This topic was discussed in [public steering meeting on 5/07](https://docs.google.com/document/d/1qazwMIHGeF3iUh5xMJIJ6PDr-S3bNkT8tNLRkSiOkOU/edit?tab=t.0) ([recording](https://youtu.be/bFXTxraccuo)) and the steering committee agreed that with [maximal representation rule](https://github.com/kubernetes/steering/blob/main/elections.md#maximal-representation), [community values favoring project over company](https://github.com/kubernetes/community/blob/master/values.md#community-over-product-or-company) and lastly with [ability to abstain from vote](https://github.com/kubernetes/steering/blob/main/charter.md#abstention) we have sufficient tools and restrictions in place to resolve any conflicts of interests.

/close
